scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -260,6 +260,26 @@ public void onFailure(Throwable caught) {
         });
     }
 
+    /**
+     * Send signal to a job
+     *
+     * @param signal signal that will be send to the job
+     * @param jobId id of the job
+     */
+    public void addJobSignal(String signal, String jobId) {
+        SchedulerServiceAsync scheduler = Scheduler.getSchedulerService();
+        scheduler.addJobSignal(LoginModel.getInstance().getSessionId(), signal, jobId, new AsyncCallback<Void>() {
+            public void onSuccess(Void result) {
+                LogModel.getInstance().logMessage("Successfully add signal " + result);
```

Comment:
what does the result return? true/false? 
if it returns a boolean so we need to add info in the log message about the signal sent + jobid

Reply:
still waiting for an answer

Reply:
This file is not updated. It returns a set of string representing the result signals

 LogModel.getInstance().logMessage("Successfully add signal " + signal + " job " + jobId +
                                                  " has the following signals " + result);